README.md

```diff
@@ -4,7 +4,7 @@ Qualitic is about developing successful quality software.
 
 ## Introduction
 
-We are committed to sharing artifacts, procedures, and best practices with each other.
+We are committed to sharing artifacts, procedures, and Possible Implementations with each other.
```

Comment:
```suggestion
We are committed to sharing artifacts, procedures, and possible Implementations with each other.
```

Reply:
It was renamed mistakenly. Please review again.